src/toxcore/dht/packet.rs

```diff
@@ -528,6 +555,16 @@ should respond with `NodesResponse` that contains up to to 4 closest nodes to
 requested public key. Ping id should be the same as it was in `NodesRequest`.
 
 https://zetok.github.io/tox-spec/#dht-packet
+
+Length    | Content
+--------- | -------------------------
+`1`       | `0x04`
+`32`      | Public Key
+`24`      | Nonce
+`1`       | Number of Response Nodes
+`[39,204]`| Packed Nodes
+`8`       | Request Id (Ping Id)
```

Comment:
Just payload will be better here instead of last 3 rows. Because current version doesn't show that payload is encrypted and has additional MACBYTES bytes.

Reply:
I got your intention.
I think that both of my docs and your intention have good explanation of packet structure.
So, I would like to hear Roman's opinion.

Reply:
I agree with Evgeniy.